cmd/login_cloud.go

```diff
@@ -14,37 +16,96 @@ var loginCloudCommand = &cobra.Command{
 	Short: "Authenticate with Load Impact",
 	Long: `Authenticate with Load Impact.
 
-This will set the default server used when just "-o cloud" is passed.`,
+This will set the default token used when just "k6 run -o cloud" is passed.`,
+
+	Example: `
+  # Show the stored token.
+  k6 login cloud -s
+
+  # Set up the token.
+  k6 login cloud -t YOUR_TOKEN
+
+  # Ask for your Load Impact user email and password to automatically set up the token.
```

Comment:
Maybe "Authenticate using your Load Impact user email and password to automatically persist an API token in configuration file on disk."?

Reply:
"Log in with an email/password"?